README.md

```diff
@@ -26,7 +26,42 @@
 The LTS Schedule is available also as a [JSON][] file or [ICal][]. There is
 also a live [Google Calendar][] that may be subscribed to.
 
-# LTS Plan
+## Mandate
+
+The Release and LTS working group's purpose is:
+
+* Management/execution the release process for all releases.
+* Maintenance of the LTS releases.
+
+Its responsibilities are:
+
+* Define the release process.
+* Generate and create releases. 
+* Manage the LTS and Current branches including backporting changes to 
+  these branches.
+
+The Release and LTS working group is structured into teams and membership in
+the working group does not automatically result in membership in these
+teams. These teams are:
+
+* Release team
+* LTS team
+* Backporting team
```

Comment:
I'm not sure I 100% agree with this. The backporting team has been trained on how to backport and has specific permissions. Not everyone who is part of the LTS working group has that